.github/workflows/mkpages.yaml

```
@@ -0,0 +1,47 @@
+name: Deploy Documentation to GitHub Pages
+
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - master
+    paths:
+      - 'docs/**'
+      - 'mkdocs.yaml'
+      - '.github/workflows/**'
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)